centraldashboard/rockcraft.yaml

```diff
@@ -65,17 +65,15 @@ parts:
       # will be built w/o flags for other architectures for simplicity.
       # Please refer to the upstream Dockerfile if that condition changes and different
       # architectures need to be supported.
-      npm rebuild && \
-      npm install && \
-      npm test && \
+      npm ci && \
       npm run build && \
       npm prune --production
 
       # install build artifacts
-      mkdir -p ${CRAFT_PART_INSTALL}/app
-      cp -r ${CRAFT_PART_SRC}/components/centraldashboard/* ${CRAFT_PART_INSTALL}/app
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/src/app
```

Comment:
Why are we changing the directory ?

Reply:
To follow what [upstream](https://github.com/kubeflow/kubeflow/blob/v1.9.0-rc.0/components/centraldashboard/Dockerfile#L27) does. This way, we will need to add minimal changes whenever switching between upstream and rock.